src/v1beta2/language_service_client_config.json

```diff
@@ -2,51 +2,51 @@
   "interfaces": {
     "google.cloud.language.v1beta2.LanguageService": {
       "retry_codes": {
+        "non_idempotent": [],
         "idempotent": [
           "DEADLINE_EXCEEDED",
           "UNAVAILABLE"
-        ],
-        "non_idempotent": []
+        ]
       },
       "retry_params": {
         "default": {
           "initial_retry_delay_millis": 100,
           "retry_delay_multiplier": 1.3,
           "max_retry_delay_millis": 60000,
-          "initial_rpc_timeout_millis": 60000,
-          "rpc_timeout_multiplier": 1.0,
-          "max_rpc_timeout_millis": 60000,
+          "initial_rpc_timeout_millis": 20000,
```

Comment:
I have just rebuilt the Docker image that makes it back to 60000. Can you just re-run `synthtool` again? These changes will go away then.

Reply:
Cool, will do